Examples/Algorithms/HepMC/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+add_library(
+  ActsExamplesProcessExtraction SHARED
```

Comment:
Usually the target name should match the module name in the filesystem. Here, it should be `ActsExamplesHepMC`.

Reply:
right